agent/php_user_instrument.c

```diff
@@ -47,11 +47,15 @@
  *
  * Many functions here call zend_bailout to continue handling fatal PHP errors,
  * Since zend_bailout calls longjmp it never returns.
+ *
+ * Note: The agent ONLY needs to call this if it has overwritten the original
+ * zend_execute_ex.
  */
 int nr_zend_call_orig_execute(NR_EXECUTE_PROTO TSRMLS_DC) {
   volatile int zcaught = 0;
   zend_try {
-    NR_PHP_PROCESS_GLOBALS(orig_execute)(NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
+    NR_PHP_PROCESS_GLOBALS(orig_execute)
+    (NR_EXECUTE_ORIG_ARGS TSRMLS_CC);
```

Comment:
Why was this reformatted? The line wasn't exceeding `ColumnLimit:     80` defined in [.clang-format](https://github.com/newrelic/newrelic-php-agent/blob/main/.clang-format#L39). I'm sorry for the nit but all formatting changes outside logic changes are very distracting.

Reply:
Oh, I see now, the parameter modified to be longer so (in another PR) so clang split, but the change didn't exist yet in this PR so it was removed, but clang didn't put it back on the same line.  While resolving conflicts, the longer param went back in so this will be split again.

Reply:
Ok. This makes sense. Thanks for taking the time to explain this!